readme.md

```diff
@@ -584,6 +585,9 @@
 - [Calculators](https://github.com/xxczaki/awesome-calculators#readme) - Calculators for every platform.
 - [Captcha](https://github.com/ZYSzys/awesome-captcha#readme) - A type of challenge–response test used in computing to determine whether or not the user is human.
 
+## Awesome related lists
+
+- [Awesome lists, search as a service](https://awesome.paulrossethings.com/) - Navigating between Awesome list has never been so easy!
```

Comment:
The title looks to be `Awesome Pretty List`.

The description should describe what it does and not be a marketing tagline.